IanGRap/Squish
# Lobby Visual Update and Fog Update

Contributors: Reed

Changes:

- LobbyStateManager.cs - Just added in some fog scripts to cleanly fade in / out the fog

- Lobby Scene - Lots of Environment assets came in, added fog into the Camera Manager (not the whole prefab just this instance) 

- Player Prefab - made the player light not effect players since it was too bright

- ColorSelector.cs - Made it work for fog

- Match Scene and Block Dunk Scene - Added color change to the fog

How to Test:

- Go through the lobby and make sure no spots are way too bright. This would be an intensity value being too high for the HDR

- Make sure no error messages pop up

- Most of the changes were just the environment gameobjects and I used a prefab to bring things over
